auxx/src/Command/Tx.hs

```diff
@@ -272,3 +273,6 @@ sendTxsFromFile diffusion txsFile = do
                 (topsortTxAuxes txAuxes)
         let submitOne = submitTxRaw diffusion
         mapM_ submitOne sortedTxAuxes
+
+fixedNM :: NetworkMagic
+fixedNM = NMNothing
```

Comment:
Why is this term repeated instead of defined once and imported?

Reply:
That is removed in a subsequent PR.

Reply:
@parsonsmatt this was defined multiple times so that we could more easily track which files had the hardcoded variable. This outlined the frontier where we needed configuration passed to.

They were removed in https://github.com/input-output-hk/cardano-sl/pull/3561